tooling/camel-tooling-model/src/main/java/org/apache/camel/tooling/model/EntityRef.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.tooling.model;
+
+/**
+ * Reference a Camel entity.
+ */
+// TODO: the org.apache.camel.catalog.Kind should probably be moved in this package/module
```

Comment:
I don't know if it would break anything but it would be useful if the `org.apache.camel.catalog.Kind` enum would be moved in the `camel-tooling-model` module

Reply:
ok you are welcome to move it

Reply:
done